pytest_django/fixtures.py

```diff
@@ -129,6 +132,16 @@ def handle_noargs(self, **options):
             patch_for_test_db_setup()
 
 
+def _disable_native_migrations():
+    from django import get_version
+
+    if get_version() > '1.7':
```

Comment:
I updated to `get_version() >= '1.7'`. The @blueyed suggestion seems to be good too. I can do it if you prefer.